rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2022-02-19"
+channel = "nightly-2022-07-24"
```

Comment:
Both in Basilisk-node and warehouse projects we use `nightly-2022-06-19`. Maybe we could also use the same here, so  we are surely on the same page for all the projects.

Reply:
Yes, we use `06-19`, but with the older version. We use `07-24` in all PRs with upgrade to polkadot `0.9.28`.